### ISSUE TITLE
Slider Multitouch

### ISSUE THREAD

**Issue by fedemaleh** [leongersen/noUiSlider#704]
Hi,
I'm using the last version of the slider and I found that if I press both nodes of the slider at the same time the slider breaks and I can't interact with anything (I can't scroll, click buttons, etc.).

**Comment by leongersen** [comment]
This may have to do with recent changes, or it may be an older issue. I'll need some more information. What device(s), what browser version? Can you try the last 8.x release and see if you have the same issue?

**Comment by DoctorOak** [comment]
I've had this issue on all touch events (so on both iOS and Android as well as Chrome's mobile inspector) when I've had two sliders that were anywhere near eachother.

I could avoid it by putting the other slider physically far away from the other one and several DOM tree levels separated (adding margins, padding or even a spacing layer between the sliders did nothing if they were within the same parent layer), but that's not actually any use so I had to remove the plugin entirely and seek alternate options.

What happens is exactly the same as what's been logged in issue #720 and it makes it impossible to use any other part of the page as a result. It effectively keeps the 'onTouchMove' event bound to the slider's handle permanently, no matter where you actually start touching the screen.

This happened to me both before and after updating the plugin to the latest version.

I hope that goes someway to helping because it's a major pity that this one issue has meant I've had to drop the use of this plugin and has caused me no end of headaches since trying to match the functionality of it.

**Comment by leongersen** [comment]
The extra information by @DoctorOak helped me narrow it down: an issue in an earlier pull request slipped through and changed -rather than resolved- a failure mode. This issue is fixed in release 9.1.0. Thanks for the report.

**Comment by nightire** [comment]
This issue may not be fixed completely, or there's at least one edge cases still suffer this.

1. when have more than one sliders
2. touch and hold / drag on one slider
3. at the same time, slide from screen edge (like back to last page) happens by accidently
4. cancel slide from screen edge, then release touching

'touchmove' event still bound to the slider's handle, if then touch another handler will trigger both of them, and it prevent the native touching / scrolling event as well.

**Comment by leongersen** [comment]
@nightire Hmm, I think is is a separate but related issue. I'd think the browser emits `touchcancel` when canceling the slide from the screen edge, which the slider doesn't explicitly handle. In a similar fashion, it has to handle `mouseout` on the document, which isn't entirely consistent across browsers. Your explanation is very clear, and I think I'd be able to reproduce this. If it is indeed `touchcancel`, it shouldn't be hard to fix.

**Comment by mbourne** [comment]
I'm reporting this because it might help to nail down a bug, or to help others.

I'm using 9.2.0 and also faced the "'touchmove' event still being bound to the slider's handle" issue on any touch device, on this page:  http://www.intmath.com/functions-and-graphs/domain-range-interactive.php

It was odd, because I didn't have the problem on this similar page: http://www.intmath.com/methods-integration/riemann-sums-negatives-discontinuities.php

One difference was the first page has "set()" directives to restrict the slider movement, whereas the second one did not.

I tried "animate: false," when creating the slider, and it now works like a charm. So it appears the "set()" with the default animate directive is still not unbinding properly when the touch event finishes.

One of the things I tried before that was to set a "padding", but that sets the same amount of padding on both sides. As a suggestion, it would be good to allow different padding amounts via an array when there is more than one slider handle involved.

Thanks for a useful plugin!
